lua/lspconfig/configs/air.lua

```diff
@@ -0,0 +1,23 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'air', 'language-server' },
+    filetypes = { 'r' },
+    root_dir = function(fname)
+      return util.root_pattern('air.toml', '.air.toml')(fname)
```

Comment:
use vim.fs.root

Reply:
`vim.fs.root` is Neovim>=0.10 while 'nvim-lspconfig' is Neovim>=0.9, though.